rustfmt.toml

```diff
@@ -1,2 +1,2 @@
-imports_granularity = "Item"
-group_imports = "One"
+imports_granularity = "Module"
+group_imports = "StdExternalCrate"
```

Comment:
Grouping imports like this I agree is good, and I was thinking of possibly doing the same.